app/views/donations/index.html.erb

```diff
@@ -43,8 +43,11 @@
                 <% end %>
                 <% if @sources.present? %>
                   <div class="form-group col-lg-3 col-md-4 col-sm-6 col-xs-12">
-                    <%= label_tag "by Source" %>
-                    <%= select_tag "filters[by_source]", options_for_select(@sources, @selected_source), {include_blank: true, class: "form-control"} %>
+                    <% id = "filter_#{SecureRandom.uuid}" %>
+                    <%= label_tag id, "Filter by Source" %>
```

Comment:
I think this changes the visible text? Was that on purpose?

Reply:
Yea. I thought it would make sense to make the labels consistent. 
![image](https://github.com/rubyforgood/human-essentials/assets/14540596/897206af-fcc1-46f2-a397-fbab41264947)
instead of 
![image](https://github.com/rubyforgood/human-essentials/assets/14540596/5530aee8-302b-4f13-bb59-c39a23f56b65)


I guess by that logic I should also change date range, but it wasn't using any of the same helpers so I didn't touch anything around there.